browser/src/Plugins/Api/LanguageClient/LanguageClientHelpers.ts

```diff
@@ -68,13 +68,14 @@ export const getTextFromContents = (contents: types.MarkedString | types.MarkedS
 
 export const bufferUpdateToTextDocumentItem = (args: Oni.BufferUpdateContext): types.TextDocumentItem => {
     const lines = args.bufferLines
-    const { bufferFullPath, filetype, version } = args.eventContext
+    const { bufferFullPath, filetype/* , modified  */} = args.eventContext
     const text = lines.join(os.EOL)
 
     return {
         uri: wrapPathInFileUri(bufferFullPath),
         languageId: filetype,
-        version,
+        // modified,
+        version: 0,
```

Comment:
Oooooooh that's what that was when I was doing this I couldn't find where we made TextDocumentItem and it makes sense because it's language-server-protocol.

Okay yeah I have a plan to fix that.